test/like.test.js

```diff
@@ -0,0 +1,312 @@
+
+import 'dotenv/config';
+import { chai, server, expect } from './test-setup';
+
+let usertoken, adminToken, mikeToken;
+
+describe('/likes and dislikes  feature', () => {
+  it('should login', (done) => {
+    chai
+      .request(server)
+      .post('/api/v1/users/login')
+      .send({
+        email: 'user@gmail.com',
+        password: 'ASqw12345'
+      })
+      .end((error, res) => {
+        if (error) done(error);
+        usertoken = `Bearer ${res.body.token}`;
+        expect(res.status).to.be.equal(200);
+        expect(res.body).to.have.deep.property('message');
+        done();
+      });
+  });
+  it('should login mike', (done) => {
+    chai
+      .request(server)
+      .post('/api/v1/users/login')
+      .send({
+        email: 'usertwo@gmail.com',
+        password: 'ASqw12345'
+      })
+      .end((error, res) => {
+        if (error) done(error);
+        mikeToken = `Bearer ${res.body.token}`;
+        expect(res.status).to.be.equal(200);
+        expect(res.body).to.have.deep.property('message');
+        done();
+      });
+  });
+  it('should login', (done) => {
```

Comment:
What's the difference between 'should login and 'should login mike'??

Reply:
yes, i need more than one user for me to test all senario.